ibm/provider/provider.go

```diff
@@ -1209,6 +1209,8 @@ func Provider() *schema.Provider {
 			"ibm_pi_spp_placement_group":             power.ResourceIBMPISPPPlacementGroup(),
 			"ibm_pi_shared_processor_pool":           power.ResourceIBMPISharedProcessorPool(),
 			"ibm_pi_workspace":                       power.ResourceIBMPIWorkspace(),
+			"ibm_pi_volumes_attach":                  power.ResourceIBMPiVolumesAttach(),
+			"ibm_pi_volumes_detach":                  power.ResourceIBMPiVolumesDetach(),
```

Comment:
I think the name should be stylized ResourceIBMPIVolumesAttach and ResourceIBMPIVolumesDetach. Please sort them with the rest of the resources.

Reply:
Done